### ISSUE TITLE
How to maintain liquid proportions

### ISSUE THREAD

**Issue by Timoti** [rachelandrew/cssgrid-ama#48]
I need to keep my templated cells square, while remaining responsive, to support the proportions of some stacked slements in a design.  I feel there must be an easy way to achieve this but can't find it.  Have tried the old vertical percentage padding thing with abso positioned content, but no dice.  What can I do?  

What I'm trying to achieve is http://snpy.in/2rRFzY, [orig for susygrid) but I end up with http://snpy.in/i5uC2E.  (There's some weird stuff going on with grid-gaps appearing  bottom and right too, but I figure one step at a time.

Absolutely loving your videos, guides etc  - appreciate all the work you've put in to making this rollout a success, and looking forward to design moving beyond a hero and three callouts.

**Comment by rachelandrew** [comment]
Can you stick up a CodePen? Otherwise this would have to wait for me to have time to build the example for you to see if I understand what you are asking, given my current workload you might wait a while!

**Comment by Timoti** [comment]
SURE! Back to you a little late on this Rachel,  as I got a big difference in rendering between my local and the codepen version, and frankly poking around at that, still I'm none the wiser.

Pen is at http://codepen.io/Timoti/pen/ZeordQ.

The utlimate project is to feature four migrant women's video interviews, and the crosshatch overlay of tightly cropped images echoes the migrant network's branding.

The design goal was to have consistant aspect ratio elements with background images, essentially tightly cropped video thumbnails. To try and tame the aspect ratio variability I switched from _background_ images to img _content_ elements and abso-positioning any internal content over the top.  At which point, proportionality improved, though differently between local - very bad (FF and chrome) against the codepen instance  - just a bit off.

You can toggle the background images at line #28 by commenting out their display:none property

Designed aspect ratios were meant to be 1:0.59

Bottom line: Looking for more consistent liquid aspect ratio rendering at different viewport sizes when there is a a contant content imge forcing minimum dimensions. Vastly preferred (I need control over proportions for other projects too): is a capability to set aspect ratios for containers of  _empty_ content.

Hope that makes sense, and thanks again.

**Comment by Timoti** [comment]
![grid proportionality issues](https://cloud.githubusercontent.com/assets/99953/24290358/8b3e61e2-10e9-11e7-81f4-f6ff76142304.jpg)
